master/buildbot/test/unit/test_scripts_cleanupdb.py

```diff
@@ -43,6 +43,19 @@ def mkconfig(**kwargs):
     return config
 
 
+def patch_environ(case, key, value):
+    """
+    Add an environment variable for the duration of a test.
+    """
+    old_environ = os.environ.copy()
+
+    def cleanup():
+        os.environ.clear()
+        os.environ.update(old_environ)
```

Comment:
what is the benefit of these two lines over simple
```python
os.environ = old_environ
```
?
`old_environ` is already a copy of the original `os.environ`.